### DISCUSSION TITLE
Create trigger to run between time windows

### DISCUSSION THREAD

**Discussion by ashah976** [quartznet/quartznet#556] [General] [answered]
Is it possible to run certain code/function between certain time windows.
Say only run function if current time is between 1pm and 2pm?

thanks
asp.net and c#

**Comment by lahma** [comment, marked as answer]
You should be able to achieve this with either [DailyTimeIntervalScheduleBuilder](http://quartznet.sourceforge.net/apidoc/2.0/html/html/c0992aec-c6a8-0701-2580-7a2395180a29.htm) or any other trigger restricted with custom calendar.
